model/project_ref.go

```diff
@@ -268,9 +270,17 @@ func (p *ProjectRef) IsDispatchingDisabled() bool {
 }
 
 func (p *ProjectRef) IsPRTestingEnabled() bool {
+	return p.IsAutoPRTestingEnabled() || p.IsManualPRTestingEnabled()
```

Comment:
just for clarification:
does that mean it would be better to something like `utility.FromBoolPtr(p.PRTestingEnabled) || utility.FromBoolPtr(p.ManualPRTestingEnabled)` rather than using the functions?

Reply:
I think this isn't relevant since both of those functions return regular booleans

Reply:
oh my bad, my brain skipped over the fact that this is a function and not just the field :see_no_evil: